docs/user_guide/installation.md

```diff
@@ -0,0 +1,293 @@
+LDAPHOST=ldap
+LDAPPORT=389
+LDAPSCHEME=ldap
+LDAPADMINPASSWORD=secret
+```
+
+Launch with:
+
+```bash
+docker-compose up -d
+```
+
+### Debugging the Gateway Container
+
+For debugging the Gateway, you can enable remote debugging by adding environment variables:
+
+```yaml
+gateway:
+  environment:
+    - JAVA_OPTIONS=-Dorg.eclipse.jetty.annotations.AnnotationParser.LEVEL=OFF -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=0.0.0.0:5005
+    - XMS=256M
+    - XMX=1G
+  ports:
+    - "5005:5005"  # Expose debug port
+```
+
+### Using Kubernetes
+
+For Kubernetes deployments, you can use the following manifest as a starting point:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: georchestra-gateway
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: georchestra-gateway
+  template:
+    metadata:
+      labels:
+        app: georchestra-gateway
+    spec:
+      containers:
+      - name: gateway
+        image: georchestra/gateway:latest
+        ports:
+        - containerPort: 8080
+        env:
+        - name: SPRING_PROFILES_ACTIVE
+          value: "docker"
+        - name: JAVA_TOOL_OPTIONS
+          value: "-Dgeorchestra.datadir=/etc/georchestra"
+        volumeMounts:
+        - name: datadir
+          mountPath: /etc/georchestra
+      volumes:
+      - name: datadir
+        persistentVolumeClaim:
+          claimName: datadir-pvc
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: georchestra-gateway
+spec:
+  selector:
+    app: georchestra-gateway
+  ports:
+  - port: 8080
+    targetPort: 8080
+  type: ClusterIP
+```
+
+## Debian Package Installation
+
+### Installing from Package
+
+A Debian package is available for installation on Debian/Ubuntu systems:
+
+```bash
+# Download the package
+wget https://packages.georchestra.org/georchestra-gateway_latest.deb
+
+# Install the package
+sudo dpkg -i georchestra-gateway_latest.deb
+
+# Install dependencies
+sudo apt-get install -f
+```
+
+### Systemd Service
+
+The Debian package includes a systemd service. You can control the service with:
+
+```bash
+# Start the service
+sudo systemctl start georchestra-gateway
+
+# Enable the service to start at boot
+sudo systemctl enable georchestra-gateway
+
+# Check the status
+sudo systemctl status georchestra-gateway
+```
+
+## JAR Installation
+
+### Building from Source
+
+To build the Gateway from source:
+
+```bash
+# Clone the repository
+git clone https://github.com/georchestra/georchestra-gateway.git
+cd georchestra-gateway
+
+# Build the project
+./mvnw clean install
+
+# The JAR file will be in gateway/target/georchestra-gateway-X.Y.Z.jar
+```
+
+### Running the JAR
+
+Run the JAR file with:
+
+```bash
+java -Dgeorchestra.datadir=/path/to/datadir -jar gateway/target/georchestra-gateway-X.Y.Z.jar
+```
+
+## Configuration
+
+After installation, configure the Gateway by editing the files in the geOrchestra data directory:
+
+1. `default.properties` - Common configuration properties
```

Comment:
not very clear. "Configuration properties common to all geOrchestra applications"

Reply:
isn't that what default.properties is in the georchestra datadir? how would you phrase it otherwise?

Reply:
"Common configuration properties" is not clear about to whom this is common. That's why I'd rephrase to "Configuration properties common to all geOrchestra applications"